supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -168,6 +168,26 @@ If referring to the program, use "Emacs", for example, "Source-Navigator support
 
 *See also*:
 
+[[executable]]
+==== image:images/yes.png[yes] Executable (adjective)
+*Description*: _Executable_ describes a file an operating system can run as a program. For directories, having executable permission allows you to navigate into the directory with the `cd` command and list the files it contains.
```

Comment:
The last part is technically not correct. If a directory contains only the x bit, then you cannot list the files in it.
Try it: 
```
$ mkdir test
$ chmod 0111      # (sets only the x bits for user/group/other, and none of the other bits)
$ ls test/
ls: cannot open directory 'test/': Permission denied
```

You set only the x bit (without r) on a directory if the path to a file in this directory is known but you don't want that a user/service/... can browse the directory (of course, this requires the r bit on the file).

If you drop "and list the files it contains" from the sentence, it's correct.

Reply:
Nice catch, @mmuehlfeldRH! Thanks!